src/database/migrations/20241122091243-alter-resource-add-assoc-res-ids.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+module.exports = {
+	up: async (queryInterface, Sequelize) => {
+		// Add the `associated_resource_ids` column to the `resources` table
```

Comment:
@adithyadinesh0412  we dont need associated_resource_ids in resource table, Vijayashree asked to create mapping table